types/wordpress__viewport/index.d.ts

```diff
@@ -13,3 +13,11 @@ declare module '@wordpress/data' {
 
 export { default as ifViewportMatches } from './if-viewport-matches';
 export { default as withViewportMatch } from './with-viewport-match';
+
+export interface ViewportStoreDescriptor extends StoreDescriptor {
+    name: 'core/viewport';
+}
+
+declare module '@wordpress/viewport' {
```

Comment:
The containing file is `@wordpress/viewport` - can't this just be `export const store: ViewportStoreDescriptor;`?

Reply:
Honestly, I'm not sure. What you are saying makes sense on the first look. However, I was following the example in #63988 - maybe @opr can chime in as to why he did it the way he did... :shrug: